src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -545,22 +537,45 @@ private void Poll()
             }
         }
 
-        private List<ConsumeResult<K, V>> PollKafka(CancellationToken token)
+        private void ProcessExceptions(Exception exception)
```

Comment:
All exceptions are routed here, we don't kill the actor on all errors anymore, we keep it alive if the Decider returns Directive.Resume.

Reply:
The side effect of this is the Decider delegate gets called twice, inside the KafkaConsumerActor and inside the stage itself.

Reply:
Only gets called twice in the event that the actor propagates the exception, right?

Reply:
For now, it is always processed twice. It can be changed so that it doesn't propagate Resume directived errors.
I always propagate it because logging is handled by the stage, I've tried to make the actor as dumb as possible and hand over all responsibilities to the stage.